packages/core/src/rpc/rpc-spec/chainHead_v1.ts

```diff
@@ -166,6 +176,48 @@ export interface StorageItemRequest {
   type: 'value' | 'hash' | 'closestDescendantMerkleValue' | 'descendantsValues' | 'descendantsHashes'
 }
 
+const PAGE_SIZE = 1000
+async function getDescendantValues(
+  block: Block,
+  params: DescendantValuesParams,
+): Promise<{
+  items: Array<{
+    key: string
+    value?: HexString
+  }>
+  next: DescendantValuesParams | null
+}> {
+  const pageSize = 100
```

Comment:
you have 2 page_size and one is 1000 and one is 100?

Reply:
my bad - leftover from previous code.

Thanks for pointing it out!